riak/tests/test_timeseries.py

```diff
@@ -213,6 +214,58 @@ def validate_data(self, ts_obj):
         self.assertEqual(row[3], 'wind')
         self.assertIsNone(row[4])
 
+    def test_query_that_creates_table_using_interpolation(self):
```

Comment:
No, because I have `{table}` in the query string, but do not use `fmt` to replace it before passing to `ts_query`. Within the code for queries, it will check for `{table}` and, if present, insert the table name.

Note that `ts_query` takes a `table` parameter than can be a `Table` object or a string. Either can provide the name of the table so it is convenient to replace the table name for the user.